modules/auxiliary/spoof/llmnr/llmnr_response.rb

```diff
@@ -99,9 +91,17 @@ def dispatch_request(packet, rhost, src_port)
           :address => (spoof.ipv6? ? spoof : spoof.ipv4_mapped).to_s
         )
       else
-        print_warning("#{rhost.to_s.ljust 16} llmnr - Unknown RR type, this shouldn't happen. Skipping")
+        vprint_warning("#{rhost.to_s.ljust 16} LLMNR - Skipping unhandled RR query type #{question.qType}")
         next
       end
+
+      if name =~ /#{datastore['REGEX']}/i
```

Comment:
I believe that is on by default, but I could add it.  Actually, thinking about the changes to this module now, I may just drop them all since they are not required for this PR -- I think I had been using ``llmnr_response`` for testing of the new LLMNR module and noticed some issues with it, but looking over it now I'm not sure what they were.